package.json

```diff
@@ -36,7 +36,7 @@
     }
   ],
   "dependencies": {
-    "react-native-tabs": "^1.0.2"
+    "react-native-tab-navigator": "joenoon/react-native-tab-navigator#master"
```

Comment:
changed here https://github.com/aksonov/react-native-router-flux/pull/701/commits/c04b03dc2c0f9bf7d3d0ab03f8b7c9bf363688a6 and in that rnrf-lock branch ive set the version to 99.0.0